docs/business-apps/introduction-to-sharepoint-business-process-integration.md

```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
 
 # SharePoint custom forms and automated business workflow
 
-Creativity and teamwork are essential to transforming business process. Most processes center on content – files and data. SharePoint gives you tools to gather and manage data in lists and libraries.  
+Creativity and teamwork are essential to transforming business processes. Most processes center on content – files and data. SharePoint offers tools to gather and manage data in lists and libraries.  
 
-Office 365 provides connections to a range of third-party systems with Microsoft Flow and PowerApps. Flow lets you easily automate business processes and PowerApps lets you create great forms and mobile apps based on SharePoint files and data. Both PowerApps and Flow feature intuitive visual designers, so anyone can craft forms and screens in an easy-to-use interface.
+Office 365 provides connections to a range of third-party systems with Power Automate and Power Apps. Power Automate lets you easily automate business processes while Power Apps lets you create exceptional forms and mobile apps based on SharePoint files and data. Both Power Automate and Power Apps feature intuitive visual designers, so anyone can build forms and screens in an easy-to-use interface.
 
-## Microsoft Flow
+## Power Automate
```

Comment:
If we change the title as suggested earlier, we could change the title of this section to "Business Process Automation".

Reply:
Change made. Please mere my branch jackwi-alt:patch2 into master if approved.